pom.xml

```diff
@@ -68,7 +68,7 @@
     <settingsPath>src/it/settings.xml</settingsPath>
     <surefire.rerunFailingTestsCount>0</surefire.rerunFailingTestsCount>
     <testcontainers.version>1.15.0</testcontainers.version>
-    <derby.version>10.14.2.0</derby.version>
+    <maria.version>2.7.0</maria.version>
```

Comment:
So we switched from derby to maria?
Still the same dependency count?

Reply:
No, we have 1 less dependency, I removed Derby. We already had mariadb.

Reply:
@janbartel  not sure we have any dependency on mariadb (except docker test for jdbc session test) and this can be removed

Reply:
The maria DB used in `org.eclipse.jetty.tests:test-jdbc-sessions` in `jetty-9.4.x` is referencing (already) ...

```
[INFO] +- org.testcontainers:mariadb:jar:1.15.0:test
[INFO] +- org.mariadb.jdbc:mariadb-java-client:jar:2.7.0:test
```

That's a tiny set of deps.

We should probably use these new properties in `test-jdbc-sessions` too.

Reply:
yup I agree on using the property or removing it.

Reply:
done